src/EventStore.Client/EventStoreClientSettings.ConnectionString.cs

```diff
@@ -195,6 +195,10 @@ private static class ConnectionStringParser {
 						connSettings.IpGossipSeeds = Array.ConvertAll(hosts, x => (IPEndPoint)x);
 				}
 
+				if (typedOptions.TryGetValue(TlsVerifyCert, out var tlsVerifyCert)) {
+					settings.ConnectivitySettings.TlsVerifyCert = (bool)tlsVerifyCert;
```

Comment:
could use the fact that we have set `settings.ConnectivitySettings.TlsVerifyCert` here and use it a few lines below in CreateHttpMessageHandler instead of getting it out of typedOptions again